plugins/kubernetes-backend/examples/dice-roller/dice-roller-manifests.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dice-roller
+  labels:
+    'backstage.io/kubernetes': dice-roller
```

Comment:
Gonna need to start discussing labels now :grin:, I'm assuming this is the service ID? If we go down that route I'm thinking `backstage.io/kubernetes-service-id`, `backstage.io/kubernetes-id`, or `kubernetes.io/service-id`, although prolly one of the former since it's not really a thing in k8s right?

Reply:
I think `backstage.io/kubernetes-id` makes sense.